crates/model/src/signature.rs

```diff
@@ -98,6 +105,19 @@ impl Signature {
                         .expect("scheme is an ecdsa scheme"),
                 )
             }
+            SigningScheme::Eip1271 => {
+                if bytes.len() < 20 {
+                    return Err(anyhow::anyhow!("The provided bytes are less than 20"));
+                }
```

Comment:
ubernit: Can be more concise with:
```suggestion
                ensure!(bytes.len() >= 20, "The provided bytes are less than 20");
```